WordPress/JetpackScreenshotGeneration/JetpackScreenshotGeneration.swift

```diff
@@ -9,13 +9,12 @@ class JetpackScreenshotGeneration: XCTestCase {
     override func setUpWithError() throws {
         super.setUp()
 
-        // Put setup code here. This method is called before the invocation of each test method in the class.
+        let app = XCUIApplication(bundleIdentifier: "com.automattic.jetpack")
 
         // This does the shared setup including injecting mocks and launching the app
-        setUpTestSuite(for: "Jetpack")
+        setUpTestSuite(for: app, removeBeforeLaunching: true)
```

Comment:
One could argue that hardcoding the bundle identifier is not a good thing to do, but I'll reply that it's no better than hardcoding the app name... 😅